DER/DerivativesContracts/Swaps.rdf

```diff
@@ -160,21 +160,51 @@
 		<fibo-fnd-utl-av:adaptedFrom xml:lang="en">ISDA Disclosure Annex for Commodity Derivative Transactions. See https://globalmarkets.bnpparibas.com/gm/features/docs/dfdisclosures/ISDA_Commodity_Derivatives_Disclosure_Annex_04_2013.pdf</fibo-fnd-utl-av:adaptedFrom>
 	</owl:Class>
 	
+	<owl:Class rdf:about="&fibo-der-drc-swp;FixedLeg">
+		<rdfs:subClassOf rdf:resource="&fibo-der-drc-swp;SwapLeg"/>
+		<rdfs:subClassOf>
+			<owl:Restriction>
+				<owl:onProperty rdf:resource="&fibo-fnd-pas-psch;hasPaymentSchedule"/>
+				<owl:onClass rdf:resource="&fibo-fnd-pas-psch;PaymentSchedule"/>
+				<owl:minQualifiedCardinality rdf:datatype="&xsd;nonNegativeInteger">0</owl:minQualifiedCardinality>
+			</owl:Restriction>
+		</rdfs:subClassOf>
+		<rdfs:label>fixed leg</rdfs:label>
+		<skos:definition>swap leg whose paying party is obligated to make payments from time to time over the term of a swap calculated by reference to a fixed annual rate or consisting of at least one payment of a fixed amount</skos:definition>
+		<fibo-fnd-utl-av:adaptedFrom xml:lang="en">2006 ISDA Definitions</fibo-fnd-utl-av:adaptedFrom>
+		<fibo-fnd-utl-av:adaptedFrom xml:lang="en">ISDA Disclosure Annex for Commodity Derivative Transactions. See https://globalmarkets.bnpparibas.com/gm/features/docs/dfdisclosures/ISDA_Commodity_Derivatives_Disclosure_Annex_04_2013.pdf</fibo-fnd-utl-av:adaptedFrom>
+	</owl:Class>
+	
 	<owl:Class rdf:about="&fibo-der-drc-swp;FixedPaymentLeg">
 		<rdfs:subClassOf rdf:resource="&fibo-der-drc-swp;SwapLeg"/>
 		<rdfs:subClassOf>
 			<owl:Restriction>
 				<owl:onProperty rdf:resource="&fibo-fnd-pas-psch;hasPaymentSchedule"/>
-				<owl:allValuesFrom rdf:resource="&fibo-fnd-pas-psch;PaymentSchedule"/>
+				<owl:someValuesFrom rdf:resource="&fibo-fnd-pas-psch;PaymentSchedule"/>
 			</owl:Restriction>
 		</rdfs:subClassOf>
 		<rdfs:label>fixed payment leg</rdfs:label>
-		<skos:definition>swap leg that specifies contractual terms associated with a schedule of payments for any swap that has a fixed leg, including, for example, a single stock swap</skos:definition>
+		<skos:definition>swap leg that specifies contractual terms associated with a schedule of payments for any swap calculated by reference to a fixed annual rate</skos:definition>
 		<fibo-fnd-utl-av:synonym>fixed payment stream terms</fibo-fnd-utl-av:synonym>
 		<fibo-fnd-utl-av:synonym>funding leg</fibo-fnd-utl-av:synonym>
 		<fibo-fnd-utl-av:usageNote>Payments may be fixed or variable, which is independent from the function of the leg (payments, return etc.). The schedule may be expressed in one of two ways: as an explicit schedule of dates or as a formula for determining payment dates in advance (taking into account for example roll rules for non working days).</fibo-fnd-utl-av:usageNote>
```

Comment:
The "or variable" is inconsistent with the definition text

Reply:
No, actually it isn't.  This is a leg whose payment schedule is fixed, but the rate can be variable.  Perhaps you missed the discussion about this in the FCT?